.github/workflows/actions/test-core-screenshot/action.yml

```diff
@@ -7,6 +7,11 @@ inputs:
     description: 'Playwright total number of test shards (ex: 4)'
   update:
     description: 'Whether or not to update the reference snapshots'
+  component:
+    description: 'The component to update the reference snapshots'
+  component-folder:
+    description: 'The component folder to update the reference snapshots'
```

Comment:
This description does not align with it's type (`boolean`). Should this match the description from `update-screenshots.yml`?

Reply:
Hopefully, the updated description is better.